src/LibVLCSharp/Platforms/Windows/VideoViewBase.cs

```diff
@@ -189,7 +204,7 @@ void CreateSwapChain()
                     BufferCount = 2,
                     SwapEffect = SwapEffect.FlipSequential,
                     Flags = SwapChainFlags.None,
-                    AlphaMode = AlphaMode.Unspecified
+                    AlphaMode = AlphaMode.Premultiplied
```

Comment:
`Straight` alpha is actually not supported in this scenario (I guess because of the pixel format used) and frames should actually come premultiplied.
The main difference between `Unspecified` and `Premultiplied` is that `Premultiplied` explicitly allows transparency, while `Unspecified` always falls back to solid color.
In my specific scenario I need the transparency because I have two layers: thumbnail below and player on top.
Videos are downloaded as you scroll through them, so if you move in between two videos, I just clear the VideoView so that the thumbnail below becomes visible until the next video is ready.

If you want to maintain the old behavior, I think that the best approach is to set `AlphaMode` to `Premultiplied`, and then to wrap the `SwapChainPanel` in the XAML in a `Border` that just binds the `Background` property from the template (that can be just set to `Black` as default)

Reply:
Ok, for the record, the error with `Straight` is the following:
```
DXGI ERROR: IDXGIFactory::CreateSwapChainForComposition: Composition SwapChains do not support the DXGI_ALPHA_MODE_STRAIGHT AlphaMode. DXGI_SWAP_CHAIN_DESC{ SwapChainType = ..._Composition, BufferDesc = DXGI_MODE_DESC1{Width = 1200, Height = 900, RefreshRate = DXGI_RATIONAL{ Numerator = 0, Denominator = 1 }, Format = B8G8R8A8_UNORM, ScanlineOrdering = ..._UNSPECIFIED, Scaling = ..._UNSPECIFIED, Stereo = FALSE }, SampleDesc = DXGI_SAMPLE_DESC{ Count = 1, Quality = 0 }, BufferUsage = 0x20, BufferCount = 2, OutputWindow = 0x0000000000000000, Scaling = ..._STRETCH, Windowed = TRUE, SwapEffect = ..._FLIP_SEQUENTIAL, AlphaMode = ..._STRAIGHT, Flags = 0x0 } [ MISCELLANEOUS ERROR #196: ]
```
There seems to be ways to fix it, but let's go with `Premultiplied` for now.